js/core/examples.js

```diff
@@ -14,7 +14,7 @@ define(
     function (pubsubhub, css) {
         var makeTitle = function (conf, $el, num, report) {
             var txt = (num > 0) ? " " + num : ""
-            ,   $tit = $("<div class='example-title'><span>Example" + txt + "</span></div>");
+            ,   $tit = $("<div class='example-title'><span>" + conf.l10n.example + txt + "</span></div>");
```

Comment:
Please avoid changing anything in the `js/` folder. We should push the changes to `conf.l10n` back to the w3c, as they would be useful there.